packages/textfield/stories/Textfield.stories.tsx

```diff
@@ -50,6 +50,12 @@ export const labelPrefix = () => (
   </TextField>
 );
 
+export const longLabelPrefix = () => (
+  <TextField className="[--w-prefix-width:90px]" value="With some value">
```

Comment:
Does this mean teams using this would have to define width if the text is longer themself and it won't happen dynamically based on content length?

Reply:
That's correct. They will only need to do so if their label is wider than the default 40px though (sounds like 1 team so far). Calculating width of an element from the component's perspective might be another temporary solution, but we didn't really have a proper testing environment to quickly investigate if such solution would work in most cases (SSR, Shadow DOM etc.). We've also already spent quite some time trying to find any quick fix for this small bug that wouldn't be a breaking change, so we felt we shouldn't spend more time on yet another temporary fix. That is also because we strongly believe this component needs a different approach, which would among other things make use of flexbox. That would however require a rewrite that we cannot afford right at this point. Let me know what you think! 😊